gui/src/app/app.component.html

```diff
@@ -29,7 +29,7 @@
           person
         </mat-icon>
         <div>
-          {{user.fullName}}
+          {{getUser().fullName}}
```

Comment:
Could you please add verification of the user not being undefined above to get rid of the warnings in console?
`<div class="user-info">` 
=>
`<div class="user-info" *ngIf="getUser() !== undefined">`

Reply:
fixed